server/src/main/java/io/spine/server/procman/ProcessManager.java

```diff
@@ -106,8 +106,17 @@ protected CommandBus getCommandBus() {
         return commandBus;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p>In {@code ProcessManager}, this method must be called from either an event applier or
```

Comment:
I think, it's in fact not an `event applier`, but `event and rejection reactors`.

That's because `ProcessManager`s do not have `@Apply` methods. They only have `@React` serving to program a reaction upon an event or a rejection.

Please update the description here and below. Probably, PR description should be updated as well.